controllers/registerController.js

```diff
@@ -25,6 +25,8 @@ exports.postRegistraionInfo = (req, res, next) => {
         // the collction will be created dynamically if it does not exist yet
         .collection('players')
         .insertOne({
+            // _id for testing duplicated-record
+            _id: "6149cebfe78b085815277716",
```

Comment:
good. 🙌🙌 but remove :)

Reply:
can i remove it after implementing the error message in FE?